tests/test_regression.py

```diff
@@ -76,6 +76,7 @@ def test_regression(boston_dataframe, return_train_score):
     )  # scale removed because build failed due to large image size
 
     # select best model
+    pycaret.regression.automl(optimize="MAPE", use_holdout=True)
```

Comment:
@Yard1 Do we already have the test case for `use_holdout = False` as well (for completeness)?